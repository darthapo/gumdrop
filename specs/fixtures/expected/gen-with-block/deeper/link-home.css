
      body {
        background: url(/logo.png);
        background: url('/logo.png');
        background: url("/logo.png");

        background: url( /logo.png );
        background: url( '/logo.png' );
      }
    