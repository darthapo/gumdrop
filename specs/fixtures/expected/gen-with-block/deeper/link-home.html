
    <a href='../../index.html'>Home</a>
    <a href="../../index.html">Home</a>
    <a href = "../../index.html">Home</a>

    <img src='../../logo.png'>
    <img src="../../logo.png">
    <img src = '../../logo.png'>

    <data data='../../logo.png'>
    <data data="../../logo.png">
    <data data = '../../logo.png'>
    